diff --git a/memcached.c b/memcached.c
index 8611661..b7a2b70 100644
--- a/memcached.c
+++ b/memcached.c
@@ -44,7 +44,9 @@
 #include <string.h>
 #include <time.h>
 #include <assert.h>
+#ifndef __MVS__
 #include <sysexits.h>
+#endif
 #include <stddef.h>
 
 #ifdef HAVE_GETOPT_LONG
@@ -61,6 +63,12 @@
 #include <sys/sysctl.h>
 #endif
 
+#ifdef __MVS__
+#define EX_USAGE 64 /* command line usage error */
+#define EX_NOUSER 67 /* addressee unknown */
+#define EX_OSERR 71 /* system error (e.g., can't fork) */
+#endif
+
 /*
  * forward declarations
  */
diff --git a/timedrun.c b/timedrun.c
index ca4eaae..6acf8fc 100644
--- a/timedrun.c
+++ b/timedrun.c
@@ -4,7 +4,12 @@
 #include <string.h>
 #include <signal.h>
 #include <sys/wait.h>
+#ifndef __MVS__
 #include <sysexits.h>
+#else
+#define EX_OSERR 71 /* system error (e.g., can't fork) */
+#define EX_SOFTWARE 70 /* internal software error */
+#endif
 
 #include <assert.h>
 
diff --git a/util.h b/util.h
index 99defd6..95b476b 100644
--- a/util.h
+++ b/util.h
@@ -1,4 +1,5 @@
 #include "config.h"
+#include <time.h>
 
 /* fast-enough functions for uriencoding strings. */
 void uriencode_init(void);
